.gitignore

```diff
@@ -56,3 +56,5 @@ target/
 *~
 *.swp
 site/
+
+.idea
```

Comment:
Actually, we often check-in the .idea directory as most of our team uses IDEA and it's config not user specific. So I can't global ignore it. 

I would see this as similar to 
```
# C extensions
*.so
```
in the current .gitingore, which clearly has nothing to do with this project, just standard stuff that tends to go in .gitignores. 

But like I said, if its bothersome I am happy to remove it or happy for you to exclude it on merge. Really, no problem at all.